feat: Update projects to compile for Visual Studio 2019 (#7)
sillsdev/SpeechAnalyzer: DEVELOPER.md

1. Obtain the source code from GitHub.  The checkout will take a while.
```bash
git clone https://github.com/sillsdev/SpeechAnalyzer
```

## Build xerces-c
The xerces-c library needs to be built for Speech Analyzer.
1. Download the xerces-c 3.1 source from https://archive.apache.org/dist/xerces/c/3/sources/xerces-c-3.1.4.zip and extract it into a folder `xerces-c\`
1. Set the Windows Environment variable `XERCES_VC10_HOME` to the full path for you have xerces-c (no trailing slash)
1. Open Visual Studio 2019 and select and open the solution file : *xerces-c\projects\Win32\VC14\xerces-all\xerces-all.sln*
1. On the menu bar, rebuild the solution for both *Debug* and *Release* configurations for *Win32*.
1. Create a `lib\` directory in xerces-c\
1. Copy the following files from xerces-c\Build\Win32\VC14\Debug\ into the created lib\ directory:
  * xerces-c_3_1D.dll
  * xerces-c_3D.lib
1. Copy the following files from xerces-c\Build\Win32\VC14\Release\ into the created lib\ directory:
  * xerces-c_3_1.dll
  * xerces-c_3.lib

Make note of these files because you will also copy them later for the Speech Analyzer build.

## Install Fonts

> Some of the following fonts in `Lib\Fonts\Musique` need to be installed for using the music features:
> * Musique Unicode.ttf
> * MusicqueU.ttf

Reviewer: I would suggest that we link directly to these TTF fonts to make it easy to download/install.

```suggestion
The following fonts need to be installed before using the music features:
* `[Lib\Fonts\Musique\Musique Unicode.ttf](https://github.com/sillsdev/SpeechAnalyzer/raw/master/Lib/Fonts/Musique/Musique%20Unicode.ttf)`
* `[Lib\Fonts\Musique\MusicqueU.ttf](https://github.com/sillsdev/SpeechAnalyzer/raw/master/Lib/Fonts/Musique/MusiqueU.ttf)`
```
